qemu/deps/performance/start_testpmd.py

```diff
@@ -94,12 +99,14 @@ def command(self, cmd):
         return to_text(self.proc.before)
 
 
-def start_testpmd(nic1_driver, nic2_driver, nic1, nic2, cores, queues):
+def start_testpmd(nic1_driver, nic2_driver, whitelist_option,
+                  nic1, nic2, cores, queues):
```

Comment:
So does here, better add it to the end when add new parameter.

Reply:
No, we can not append it at the end of the command. it must be before "--".

Reply:
Also, `start_testpmd(nic1_driver, nic2_driver, nic1, nic2, cores, queues, whitelist_option)` is better and does not harm what you want to implement. But still, I am fine.